econml/ortho_forest.py

```diff
@@ -305,12 +344,17 @@ def _fit_forest(self, Y, T, X, W=None):
                 self.min_leaf_size, self.max_depth,
                 self.random_state.randint(MAX_RAND_SEED)) for s in subsample_ind)
 
-    def _get_weights(self, X_single):
-        # Calculates weights
+    def _get_weights(self, X_single, tree_slice=None):
+        # Calculates weights for an input feature vector for a slice of trees
+        # defined by the `tree_slice` tuple (start, end)
```

Comment:
Is it obvious that start is inclusive and end is exclusive?

(not a rhetorical question - maybe in python this is obvious)

Reply:
Fixed the comments!